int32 drone_id
int32 observe_counter
geometry_msgs/Vector3[] vectors